keyboards/handwired/dactyl_manuform/6x7/keymaps/iso-de/readme.md

```diff
@@ -0,0 +1,15 @@
+# 6x7 Dactyl Manuform ISO-DE
+
+Hardware build log: https://github.com/andi4000/dactyl-keyboard
+
+Please note the wiring differences with the default keymap in `config.h`.
+
+## Layout
+
+![Dactyl Manuform 6x7 ISO-DE](https://i.imgur.com/LUlCGVd.png)
+
+## Flashing
+```bash
+qmk flash -kb handwired/dactyl_manuform/6x7 -km iso-de
+```
+Reset the keyboard when prompted to enter bootloader.
```

Comment:
```suggestion
Hardware build log: [andi4000/dactyl-keyboard](https://github.com/andi4000/dactyl-keyboard)

Please note the wiring differences with the default keymap in `config.h`.

## Layout

![Dactyl Manuform 6x7 ISO-DE](https://i.imgur.com/LUlCGVdh.png)

## Flashing
```bash
qmk flash -kb handwired/dactyl_manuform/6x7 -km iso-de
```

Reset the keyboard when prompted to enter bootloader.
```

Reply:
Thanks for the suggestion! I have updated the formatting on the readme.